package.json

```diff
@@ -28,8 +28,8 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.3.4",
     "phantomjs": "^1.9.19",
-    "react": "^15.0.1",
-    "react-dom": "^15.0.1",
+    "react": "~15.0.1",
```

Comment:
I fixed this issue in Master. Basically I added `react-addons-test-utils` as a dev-dependency. We already rely on both react and react-dom for tests so it was fairly straightforward. 

Can you please change this back?